src/kirby/components/list/list.component.ts

```diff
@@ -75,13 +75,29 @@ export class ListComponent implements OnInit, OnChanges {
    * - list, if {@link #isSectionsEnabled} is `false`
    * - section, if {@link #isSectionsEnabled} is `true`
    *
-   * `square` means **without** rounded corners, `rounded` means **with** rounded corners.
+   * `square` means **without** rounded corners, `rounded` means **with** rounded corners.,  `none` means **without** padding, border, box-shadow and background.
    */
   @Input() shape: ListShape = 'rounded';
   @HostBinding('class.rounded')
   public get isRounded(): boolean {
     return this.shape === 'rounded';
   }
+  @HostBinding('class.no-shape')
+  public get isNoShape(): boolean {
+    return this.shape === 'none';
+  }
+
+  /**
+   * Determine spacing between list items:
+   * - list, if {@link #isSectionsEnabled} is `false`
+   *
+   * `true` means **with** space between list items, `false` means **without** any space between list items.
+   */
+  @Input() itemSpacing: boolean = false;
+  @HostBinding('class.spaced')
```

Comment:
If css class is applied on entire list consider naming according to items:
```suggestion
  @HostBinding('class.separated-items')
```
OR
```suggestion
  @HostBinding('class.space-between-items')
```